db/indexes.go

```diff
@@ -50,6 +53,10 @@ const (
 	IndexAllDocs
 	IndexTombstones
 	IndexSyncDocs
+	IndexSyncUser
```

Comment:
IndexUsers, IndexRoles, IndexSessions are better naming for these (as there's no concept of non-sync users/roles/sessions)

Reply:
make sense. updated

Reply:
@adamcfraser do you want to update `indexNames` for these to not include `sync` as well? I chose to keep sync in it since we use `\\_sync:role:%` 

the change will be something like below:

```golang
indexNames = map[SGIndexType]string{
// other indexes
	IndexUser:       "users",
	IndexSession:    "sessions",
	IndexRole:       "roles",
}
```